onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3109,10 +3110,19 @@ def aten_msort(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_mul(self: TensorType, other: TensorType) -> TensorType:
+def aten_mul(self, other) -> TensorType:
     # mul.Tensor(Tensor self, Tensor other) -> Tensor
 
-    raise NotImplementedError()
+    return op.Mul(self, other)
+
+
+def aten_mul_bool(self: BOOL, other: BOOL) -> BOOL:
```

Comment:
`BOOL[...]` instead of `BOOL` ? Unless the type-annotation-design is going to change the meaning of `BOOL`, which is currently a tensor of rank-0

Reply:
yeah but mypy complains about BOOL[...]

Reply:
I’ll see if I can turn it off

Reply:
Couldn't make it to work. I will leave it as is for now as update when things fit together. Currently the types for the ops are also BOOL etc. I think?